Policies/github-community-guidelines.md

```diff
@@ -10,110 +10,75 @@ topics:
-
-  To facilitate a path to abuse resolution with project maintainers themselves, prior to escalation to GitHub abuse reports, we recommend, but do not require, that repository owners take the following steps when posting potentially harmful security research content:
-
-    * Clearly identify and describe any potentially harmful content in a disclaimer in the project’s README.md file or source code comments.
-    * Provide a preferred contact method for any 3rd party abuse inquiries through a SECURITY.md file in the repository (e.g. "Please create an issue on this repository for any questions or concerns"). Such a contact method allows 3rd parties to reach out to project maintainers directly and potentially resolve concerns without the need to file abuse reports.
-
-  *GitHub considers the npm registry to be a platform used primarily for installation and run-time use of code, and not for research.*
-
-
-## What happens if someone breaks the rules?
-
-There are a variety of actions that we may take when a user reports inappropriate behavior or content. It usually depends on the exact circumstances of a particular case. We recognize that sometimes people may say or do inappropriate things for any number of reasons. Perhaps they did not realize how their words would be perceived. Or maybe they just let their emotions get the best of them. Of course, sometimes, there are folks who just want to spam or cause trouble.
-
-Each case requires a different approach, and we try to tailor our response to meet the needs of the situation that has been reported. We'll review each abuse report on a case-by-case basis. In each case, we will have a diverse team investigate the content and surrounding facts and respond as appropriate, using these guidelines to guide our decision.
+## Appeal and Reinstatement
 
-Actions we may take in response to an abuse report include but are not limited to:
+In some cases there may be a basis to reverse a moderation action taken by GitHub Staff. 
 
-* Content Removal
-* Content Blocking
-* Account Suspension
-* Account Termination
+* **Reinstatement**: Where a user wishes to address the violation and is willing to agree to abide by our Acceptable Use Policies moving forward, we may choose to reinstate their account or content depending on the severity of the initial violation. 
 
-## Appeal and Reinstatement
+* **Appeal**: If a user wishes to dispute the basis of an enforcement action and can provide additional information regarding the alleged violation, we will review that information and may grant the appeal where we determined that a violation did not occur.
```

Comment:
Who reviews appeals?

Reply:
> Who reviews appeals?

Please see https://docs.github.com/github/site-policy/github-appeal-and-reinstatement to learn about how we handle appeals.